.changeset/twenty-cobras-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lynx-js/rspeedy": minor
```

Comment:
Since we are still in 0.x stage, we usually use `patch` for features and `minor` for BREAKING CHANGEs.
```suggestion
"@lynx-js/rspeedy": patch
```